### PR TITLE
add command line option of setting cold calls

### PR DESCRIPTION
**Pull request by wbgilmartin**
add parameter to the rocblas-bench to set the number of cold calls used for bench marking. 

this feature is enabled for the gemm tests.

### CODE REVIEW

File: clients/include/rocblas_common.yaml [hunk 1]
```diff
@@ -258,6 +258,7 @@ Arguments:
   - unit_check: rocblas_int
   - timing: rocblas_int
   - iters: rocblas_int
+  - colde_iters: rocblas_int
```
Comment (leekillough) [inline clients/include/rocblas_common.yaml]: misspelling

File: clients/include/testing_gemm_strided_batched.hpp [hunk 2]
```diff
@@ -238,7 +238,7 @@ void testing_gemm_strided_batched(const Arguments& arg)
 
     if(arg.timing)
     {
-        int number_cold_calls = 2;
+        int number_cold_calls = arg.iters;
```
Comment (leekillough) [inline clients/include/testing_gemm_strided_batched.hpp]: should be `cold_iters`

Comment (leekillough) [reply clients/include/testing_gemm_strided_batched.hpp]: Shouldn't that be `arg.cold_iters`, or is it a special case?

Comment (wbgilmartin) [reply clients/include/testing_gemm_strided_batched.hpp]: fixed in last commit

**Pull request merged**